client/my-sites/plans/ecommerce-trial/ecommerce-trial-expired/index.tsx

```diff
@@ -50,6 +52,27 @@ const ECommerceTrialExpired = (): JSX.Element => {
 		} );
 	}, [] );
 
+	const plansContent = isEnabled( 'plans/wooexpress-small' ) ? (
+		<WooExpressPlans
+			interval={ interval }
+			monthlyControlProps={ monthlyControlProps }
+			siteId={ siteId ?? 0 }
+			yearlyControlProps={ yearlyControlProps }
+		/>
+	) : (
+		<ECommercePlanFeatures
+			interval={ interval }
+			monthlyControlProps={ monthlyControlProps }
+			planFeatureSets={ expiredTrialWooExpressMediumPlanFeatureSets }
+			priceCardSubtitle={ translate(
+				'Kickstart your growth with the world’s most-trusted ecommerce platform.'
+			) }
+			siteSlug={ siteSlug }
+			triggerTracksEvent={ triggerTracksEvent }
```

Comment:
Should we have Tracks events for the Woo Express screens as well (or do we have a card for that for later?)